### ISSUE TITLE
Some questions about the loss

### ISSUE THREAD

**Issue by FeiWard** [hellochick/PSPNet-tensorflow#15]
When I use this code to train the cityscape as the paper say. But I found that the loss is always at about 2~~3. I want to know if this is an error or BUG?

**Comment by FeiWard** [comment]
I have decide this issue after I use the pretrained model.

**Comment by ningscapr** [comment]
Hi, i have the same problem. Which pretrained model did you use that you sloved this issue? @FeiWard

**Comment by codermckee** [comment]
> Hi, i have the same problem. Which pretrained model did you use that you sloved this issue? @FeiWard

I also met this problem. the loss never decrease.

**Comment by codermckee** [comment]
update. Now I use the cityscapes-model as pretrained model to train mapillary project of 66 classes. the loss  dropped to 0.1～2.
